### DISCUSSION TITLE
Need urgent help(unable to run)

### DISCUSSION THREAD

**Discussion by Vineet-23** [ghiscoding/slickgrid-react#122] [Q&A]
Hi there, 

I am trying to run but getting error as 
"Could not resolve multiple-select-vanilla/dist/styles/sass/multiple-select.scss: Could not resolve multiple-select-vanilla/dist/styles/sass/multiple-select.scss
    ╷
910 │ @import 'multiple-select-vanilla/dist/styles/sass/multiple-select.scss';
    │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ╵
  node_modules/@slickgrid-universal/common/dist/styles/sass/_variables.scss 910:9               @import
  node_modules/@slickgrid-universal/common/dist/styles/sass/slick-editors.scss 1:9              @import
  node_modules/@slickgrid-universal/common/dist/styles/sass/slickgrid-theme-bootstrap.scss 9:9  @import
  stdin 12:9                                                                                    root stylesheet"

someone please help me to fix this issue.

Thanks in advance 
Vineet

**Comment by ghiscoding** [comment]
Multiple-Select-Vanilla is a new dependency of Slickgrid-Universal v3.0 that should get installed and its styles is published correctly as seen on [unpkg](https://unpkg.com/browse/multiple-select-vanilla@0.3.1/dist/styles/), you should check your `node_modules` to see if it is installed and check if that folder location exists. It also seems that you're using SASS since you're getting an scss file error, you could try to load the CSS file instead to see if that works. You could also compare against the [Slickgrid-React-Demo](https://github.com/ghiscoding/slickgrid-react-demos/blob/main/bootstrap5-i18n-demo/src/slickgrid.scss), you barely provided any info to help.

Also note that anytime someone writes "urgent help" (even worst in uppercase), it has the contrary effect on me and I don't even want to help or I just wait a long time before replying and helping. Because seriously this is a free open project provided to anyone for free and I don't work for you! So please keep that in mind

**Reply by Vineet-23** [reply]
First of all thanks for replying. 
I was trying to fix the error since past 2 days but couldn't so finally posted here. 
I apologize for the posting my issue. I should understand that since its a free open project so shouldn't expect help if facing any issues. 
Will keep it in mind hence and thanks once again. 
God bless you !

**Reply by ghiscoding** [reply]
I think your best option is to try the Slickgrid-React-Demo that I provided the link, just clone it and run it, then from there you can compare with your project. The fact is that I don't even use React, this port was created by another user (copied from my other Angular-Slickgrid port) and eventually we got it all working, so anyway I can't help much on React stuff but the demo is working nicely so you should really try that first by simply cloning and running it. If you should have problem then you need to provide a reproduction sample stackblitz or a github project
